src/lib/style/abstracts/_variables.scss

```diff
@@ -1,7 +1,29 @@
+@use 'sass:color';
+@use 'sass:map';
+
 // -----------------------------------------------------------------------------
 // This file contains all application-wide Sass variables.
 // -----------------------------------------------------------------------------
 
+// Colors defined by design: https://www.figma.com/file/0oPm5lLSUva8kyfVNMS6FA/Synapse-Style-%26-Component-Library?node-id=77%3A1232
+$colors: (
+  'gray-1000': #22252a,
+  'gray-900': #353a3f,
+  'gray-800': #4a5056,
+  'gray-700': #878e95,
+  'gray-600': #aeb5bc,
+  'gray-500': #d0d4d9,
+  'gray-400': #dfe2e6,
+  'gray-300': #eaecee,
+  'gray-200': #f1f3f5,
+  'gray-100': #fbfbfc,
+  'white': #ffffff,
+  'success': #32a330,
+  'info': #017fa5,
+  'warning': #cc9f00,
+  'error': #c13415,
+) !default;
```

Comment:
There is still a lot of work to do to use these colors rather than the variables below.